images/builder/main.go

```diff
@@ -67,7 +67,15 @@ func getImageName(o options, tag string, config string) (string, error) {
 	if err := yaml.Unmarshal(buf, &cloudbuildyamlFile); err != nil {
 		return "", fmt.Errorf("failed to get image name: %v", err)
 	}
-	var projectID, _ = getProjectID()
```

Comment:
always assuming we should discover it from the current project is wrong when we have most builds run in another project ...